lib/ansible/modules/windows/win_updates.py

```diff
@@ -49,6 +49,18 @@
         - Tools
         - UpdateRollups
         - Updates
+    post_category_names:
```

Comment:
Is there an easy way for someone to see the categories assigned to an update? Does the update catalogue easily show this?

Reply:
From what I can see the Microsoft Update catalog doesn't directly provide these strings which would be very useful. Running the module in search mode will return the list of category strings once this patch is applied independent of post-category matching.

Reply:
Added